sysdig/resource_sysdig_secure_organization.go

```diff
@@ -0,0 +1,151 @@
+package sysdig
+
+import (
+	"context"
+	v2 "github.com/draios/terraform-provider-sysdig/sysdig/internal/client/v2"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"time"
+)
+
+func resourceSysdigSecureOrganization() *schema.Resource {
+	timeout := 5 * time.Minute
+
+	return &schema.Resource{
+		CreateContext: resourceSysdigSecureOrganizationCreate,
+		DeleteContext: resourceSysdigSecureOrganizationDelete,
+		ReadContext:   resourceSysdigSecureOrganizationRead,
+		UpdateContext: resourceSysdigSecureOrganizationUpdate,
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+		Timeouts: &schema.ResourceTimeout{
+			Create: schema.DefaultTimeout(timeout),
+			Update: schema.DefaultTimeout(timeout),
+			Read:   schema.DefaultTimeout(timeout),
+			Delete: schema.DefaultTimeout(timeout),
+		},
+		Schema: map[string]*schema.Schema{
+			"cloud_provider_id": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"cloud_provider_type": {
+				Type:     schema.TypeInt,
+				Required: true,
```

Comment:
Should we add validation here to restrict this to just supported providers?

Reply:
done!